deployments/warp_routes/FORM/ethereum-form-config.yaml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+tokens:
```

Comment:
do we need to add logouri fields so that the image pops up?

Reply:
+1

Reply:
done